assets/js/subscribe_widget.js

```diff
@@ -28,30 +32,33 @@ function subscribe(retries = 5) {
     subscribe_button.innerHTML = '<b>' + subscribe_data.unsubscribe_text + ' | ' + subscribe_data.sub_count_text + '</b>';
 
     xhr.onreadystatechange = function () {
-        if (xhr.readyState == 4) {
-            if (xhr.status != 200) {
+        if (xhr.readyState === 4) {
+            if (xhr.status !== 200) {
                 subscribe_button.onclick = subscribe;
                 subscribe_button.innerHTML = fallback;
```

Comment:
Shouldn't that use `.textContent` too?

Reply:
No, in this case `fallback` is [portion of HTML](https://github.com/iv-org/invidious/blob/70cbe91776d1de10f2767c6a5ad5912fd705bdd3/assets/js/subscribe_widget.js#L26) 😸

Reply:
Ah, okay, my bad ^^